book/installation/docker.md

```diff
@@ -83,20 +83,33 @@ To run Reth with Docker Compose, run the following command from a shell inside t
 docker compose -f etc/docker-compose.yml up -d
 ```
 
+If you want to also run Lighthouse, run the following command instead:
```

Comment:
Add a note here like

```
> **Note**
>
> If you want to run without Lighthouse, the JWT for the consensus client is in ..., and the engine API is on ...
```

To explain how to get the JWT for your own CL setup (and where reth's engine API is)